dev/isort.cfg

```diff
@@ -1,9 +1,9 @@
 # Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
+# contributor license agreements. See the NOTICE file distributed with
```

Comment:
Hm, really, it guesses at moves? OK, if the net effect is that `tox.ini` has been deleted, OK. If you wouldn't mind, if you make another pass here, just add back those spaces. They seem to be present in the 'canonical' text of the license and best to just avoid any minor deviation regardless of where it came from.

So the `lint-python` script still works with this I take it because I see you updated it, and that's the important thing.